README.md

```diff
@@ -0,0 +1,23 @@
+# Doula Cooperative of Rochester
+ 
+ Site: https://www.doulacooperative.org
+
+ ## Getting Started
+ ```
+ $ npm run tools
+ $ firebase login
+ $ yarn install
+ $ yarn serve
+ ```
+
+ ## Modify the Doula List
+ 1. Update: src/templates/partials/_doulas.pug
+ 2. Ensure any new profiles have a corresponding jpg and webp file in src/img/headshots (the filename needs to match the name key of the partial). Photos must be 200p wide by 280p tall (see: the-doulas.pug)
```

Comment:
Yeah......I do some pretty intense image editing and conversion to get the images in the right format. Conversion is done with the webp.js file, but the editing involves some art direction, cropping, and creating two sizes. There's a headshot, and then a headshot thumbnail.

I can produce these specs, but I'm not sure this process can/should be automated. Thoughts?

Reply:
Let's hold off on user-contributed images til there's a need.